CHANGELOG.md

```diff
@@ -20,4 +23,4 @@
 
 ## 0.1.0
 
-* Initial Release
\ No newline at end of file
+* Initial Release
```

Comment:
no idea what's up, I edited in the GitHub UI. probably added a trailing newline

Reply:
IIRC it does a trailing newlines by default.